.github/workflows/ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       matrix:
         php-version: [
             '7.4',
-            '8.1'
+            '8.2'
```

Comment:
How it's related to symfony update?

Reply:
@asmarovydlo this changes make by @dereuromark. Looks like for check validation 
https://github.com/spryker/architecture-sniffer/pull/117/commits/c2e4275f2d4da4ea76a0d9e3ab67c838816b73ef

Reply:
@dereuromark we have a separate ticket for updating to 8.2, do we really want to make it here?

Reply:
Both is fine

Reply:
As CI already teils u